Add select a node before migration (#345)
harvester/docs: docs/vm/live-migration.md

![](/img/v1.2/vm/migrate-action.png)

> When you have [node scheduling rules](https://docs.harvesterhci.io/v1.1/vm/create-windows-vm/#node-scheduling-tab) configured for a VM, you must ensure that the nodes you are migrating to meet the VM's runtime requirements. The list of nodes you get to search and select from will be generated based on:

Reviewer: ```suggestion
When you have [node scheduling rules](https://docs.harvesterhci.io/v1.1/vm/create-windows-vm/#node-scheduling-tab) configured for a VM, you must ensure that the target nodes you are migrating to meet the VM's runtime requirements. The list of nodes you get to search and select from will be generated based on:
```

### Progress Timeout

Live migration will also be aborted when copying memory doesn't make any progress in 150s.

## Select Migratable Nodes

> When users have configured Node scheduling rules for a VM (**[Node Scheduling Tab](https://docs.harvesterhci.io/v1.1/vm/create-windows-vm/#node-scheduling-tab)**), ensure that the migrated nodes meet the VM's runtime requirements. When selecting the Nodes list before migration, the Nodes will be searched based on the scheduling rules parameters of the VM.

Reviewer: ```suggestion
When you have [node scheduling rules](https://docs.harvesterhci.io/v1.1/vm/create-windows-vm/#node-scheduling-tab) configured for a VM, you must ensure that the nodes you are migrating to meet the VM's runtime requirements. The list of nodes you get to search and select from will be generated based on:
  - VM scheduling rules.
  - Possibly node rules from the NetworkConfig.
```

  - VM scheduling rules.

>   - Possibly node rules from the NetworkConfig.

Reviewer: ```suggestion
  - Possibly node rules from the network configuration.
```

> These Nodes are filtered based on the VM scheduling rules, and it's worth noting that the VM may include Node rules from the NetworkConfig.

Reviewer: ```suggestion
```
